AdobeStockAdminUi/Test/Unit/Model/IsAdobeStockIntegrationEnabledTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeStockAdminUi\Test\Unit\Model;
+
+use Magento\AdobeStockAdminUi\Model\IsAdobeStockIntegrationEnabled;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test functionality which is used for identification is the Adobe Stock integration enabled or not
+ */
+class IsAdobeStockIntegrationEnabledTest extends TestCase
+{
+    private const XML_PATH_ENABLED = 'adobe_stock/integration/enabled';
```

Comment:
This value is used basically to identify to add or not "Search Adobe Stock" button to media gallery. The best place for it is `AdobeStockImageAdminUi` module

Reply:
Ok, got it. Will change.